### PR TITLE
[base] Vaccine cims base

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard10**
this base PR includes features which has already been reviewed in other PR's:
- updated vaccine introduction
- vaccinated people chart

Todo:
- communicate new translation keys with communication
- remove the checks for the currently optional `vaccine_coverage` data

### CODE REVIEW

**Review by VWSCoronaDashboard1 — APPROVED** [review]

**Review by VWSCoronaDashboard8 — APPROVED** [review]

File: packages/app/src/static-props/get-data.ts [hunk 1]
```diff
@@ -91,7 +91,7 @@ export function selectNlData<T extends keyof National = never>(
     const { data } = getNlData();
 
     const selectedNlData = metrics.reduce(
-      (acc, p) => set(acc, p, data[p]),
+      (acc, p) => set(acc, p, data[p] ?? null),
```
Comment (VWSCoronaDashboard8) [inline packages/app/src/static-props/get-data.ts:94]: Looks like this is setting null for data that is undefined, but optional data according to the schema's should be simply undefined, no?

**Review by VWSCoronaDashboard8 — APPROVED** [review]

File: packages/app/src/domain/topical/topical-vaccine-tile.tsx [hunk 2]
```diff
@@ -51,6 +50,13 @@ export function TopicalVaccineTile({ data }: TopicalVaccineProps) {
       <Text mt={0}>
         {replaceComponentsInText(text.administered_tests, {
           administeredVaccines: <strong>{formatNumber(estimated)}</strong>,
+          vaccine_coverage: data.vaccine_coverage ? (
```
Comment (VWSCoronaDashboard4) [inline packages/app/src/domain/topical/topical-vaccine-tile.tsx:53]: nit: it's a bit ugly to have one variable in the string is camel cased and the other snake cased.

File: packages/app/src/static-props/get-data.ts (hunk 1 above)
Comment (VWSCoronaDashboard1) [reply packages/app/src/static-props/get-data.ts]: Replaced by pickBy

**Review by VWSCoronaDashboard8 — APPROVED** [review]

File: packages/app/src/static-props/get-data.ts [hunk 3]
```diff
@@ -180,9 +185,12 @@ export function selectGmData<T extends keyof Municipal = never>(
...
-    const selectedGmData = metrics.reduce(
-      (acc, p) => set(acc, p, gmData.data[p]),
-      {} as Pick<Regionaal, T>
+    const selectedGmData = pickBy(
+      metrics.reduce(
+        (acc, p) => set(acc, p, gmData.data[p]),
+        {} as Pick<Regionaal, T>
+      ),
+      isDefined
     );
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/static-props/get-data.ts:194]:
```suggestion
const selectedGmData = metrics.reduce(
  (acc, p) => isDefined(gmData.data[p]) ? set(acc, p, gmData.data[p]) : acc,
  {} as Pick<Regionaal, T>
);
```

File: packages/app/src/static-props/get-data.ts [hunk 4]
```diff
@@ -130,9 +132,12 @@ export function selectVrData<T extends keyof Regionaal = never>(
...
-    const selectedVrData = metrics.reduce(
-      (acc, p) => set(acc, p, vrData.data[p]),
-      {} as Pick<Regionaal, T>
+    const selectedVrData = pickBy(
+      metrics.reduce(
+        (acc, p) => set(acc, p, vrData.data[p]),
+        {} as Pick<Regionaal, T>
+      ),
+      isDefined
     );
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/static-props/get-data.ts:141]:
```suggestion
    const selectedVrData = metrics.reduce(
      (acc, p) => isDefined(vrData.data[p]) ? set(acc, p, vrData.data[p]) : acc,
      {} as Pick<Regionaal, T>
    );
```

File: packages/app/src/static-props/get-data.ts [hunk 5]
```diff
@@ -90,9 +92,9 @@ export function selectNlData<T extends keyof National = never>(
   return () => {
     const { data } = getNlData();
 
-    const selectedNlData = metrics.reduce(
-      (acc, p) => set(acc, p, data[p]),
-      {} as Pick<National, T>
+    const selectedNlData = pickBy(
+      metrics.reduce((acc, p) => set(acc, p, data[p]), {} as Pick<National, T>),
+      isDefined
     );
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/static-props/get-data.ts:98]:
```suggestion
    const selectedNlData = metrics.reduce(
      (acc, p) => isDefined(data[p]) ? set(acc, p, data[p]) : acc,
      {} as Pick<National, T>
    );
```

**Review by VWSCoronaDashboard10 — COMMENTED** [review]
I think we can go without pickBy

**Pull request merged**